gui/default/syncthing/core/syncthingController.js

```diff
@@ -1670,7 +1670,19 @@ angular.module('syncthing.core')
 
             $scope.config.folders = folderList(folderListCache);
             $scope.saveConfig();
-        }
+        };
+
+        $scope.getAllFolderPauseState = function(pause) {
+            var folderListCache = $scope.folderList();
+
+            for (var i = 0; i < folderListCache.length; i++) {
+                if (folderListCache[i].paused == pause) {
+                    return true;
```

Comment:
Also from the naming convention maybe the method should start with is/are such as areAllFolderPausedStateSetTo(), but don't care much about this.